.github/workflows/go.yml

```diff
@@ -18,21 +18,21 @@ jobs:
         image: mysql:latest
         ports:
           - 3306:3306
-        options: >
-          --health-cmd="my"sqladmin ping"  --health-interval=10s --health-timeout=5s --health-retries=5
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=5
         env:
-          MYSQL_PASSWORD: ${{ secrets.MYSQL_PASSWORD }}
+          MYSQL_ROOT_PASSWORD: goodAhSQLPassword!
```

Comment:
I would prefer these be hidden passwords, and not committed. However, since these are throw away passwords and we will be adding the functionality for hidden passwords for now this is okay.